### ISSUE TITLE
error while extracting features via pretrained resnet-200

### ISSUE THREAD

**Issue by DelightRun** [facebook/fb.resnet.torch#45]
When I extract features by execute:

```
th extract-features.lua resnet-200.t7 /path/to/my/image
```

I got:

```
In 5 module of nn.Sequential:
In 1 module of nn.Sequential:
In 1 module of nn.Sequential:
In 1 module of nn.ConcatTable:
In 7 module of nn.Sequential:
...09/torch/install/share/lua/5.1/nn/BatchNormalization.lua:80: got 64-feature tensor, expected 256
```

But `resnet-152.t7` is OK, so what's the problem?

**Comment by Cysu** [comment]
There was a typo in the preresnet model as shown in #41 and was fixed in #42. Thus the pretrained resnet-200.t7 does not match the current model structure.

You may temporarily switch back to the previous version to use the pretrained model.

`git checkout a446597550f7e4b15866e07b32c21e63cd45064b -b for-pretrained-resnet200`

**Comment by DelightRun** [comment]
All right, thank you.

**Comment by colesbury** [comment]
I modified the `resnet-200.t7` to fix the batch norms. If you download it again it should work now.
